cosmoz-moment.html

```diff
@@ -12,10 +31,10 @@
 		window.Cosmoz = window.Cosmoz || {};
 
 		var localSharedMoment = moment(),
-			locale = 'en',
```

Comment:
Is it correct indent: 

    `var localSharedMoment = moment(),
     locale = 'en',
     momentElements = [];` 

or you want this:

    `var localSharedMoment = moment(),
		 locale = 'en',
		 momentElements = [];` 

Now we have first indent for variables